src/content/docs/cypher/query-clauses/load-from.md

```diff
@@ -144,7 +144,20 @@ You can also see the details of any warnings generated by the skipped lines usin
 See the [ignoring erroneous rows section of `COPY FROM`](import#ignore-erroneous-rows) for more details.
 
 ## Scan Data Formats
-Load from can scan several raw or in-memory file formats, such as CSV, Parquet, Pandas, Polars, Arrow tables, and JSON.
+`LOAD FROM` can scan several raw or in-memory file formats, such as CSV, Parquet, Pandas, Polars, Arrow tables, and JSON.
+
+### File format detection
```

Comment:
I think this is also available for COPY. I would try to organize the options we support for `LOAD FROM` and `COPY`. Though this can be done after the release.